account_invoice_shipping_address/__openerp__.py

```diff
@@ -23,9 +23,9 @@
 
 {
     "name": "Invoice Shipping Address",
-    'summary': """
+    'description': """
```

Comment:
Don't put description. The description is in the README.rst file. This is only summary.